src/components/UpcomingEventsList.tsx

```diff
@@ -2,14 +2,13 @@ import { useEffect, useState } from "react"
 import _ from "lodash"
 import { useRouter } from "next/router"
 import { useTranslation } from "next-i18next"
-import { Box, Grid, Heading } from "@chakra-ui/react"
 
 import type { CommunityConference, Lang } from "@/lib/types"
 
-import { Button } from "@/components/Buttons"
 import EventCard from "@/components/EventCard"
 import InfoBanner from "@/components/InfoBanner"
 import InlineLink from "@/components/Link"
```

Comment:
Wasn't sure what to do here, assuming we'll migrate this separately but might be wrong as not on list.

Reply:
We should use the Link that is in `components/ui/Link`. Not a big deal though.

Reply:
Okay, replaced it